src/main/kotlin/kweb/WebBrowser.kt

```diff
@@ -91,29 +95,53 @@ class WebBrowser(private val sessionId: String, val httpRequestInfo: HttpRequest
         return JSFunction(stringBuilder.toString(), params.joinToString(separator = ","))
     }
 
+    private fun argumentsToJsonElement(args: Array<out Any?>) : List<JsonElement>{
+        val argList = mutableListOf<JsonElement>()
+        args.forEach {
+            if (it == null) {
+                argList.add(JsonNull)
+            } else {
+                when (it) {
+                    is String, is Int, is Boolean, is Float, is Double,
```

Comment:
Should we be using https://kotlin.github.io/kotlinx.serialization/kotlinx-serialization-json/kotlinx-serialization-json/kotlinx.serialization.json/encode-to-json-element.html here?